.github/pull_request_template.md

```diff
@@ -27,3 +29,42 @@ Major changes require a number of considerations including impact on:
 * Elasticsearch Service (https://cloud.elastic.co)
 * Elastic Cloud Enterprise (https://www.elastic.co/products/ece)
 -->
+
+## Checklist
+<!--
+Add a checklist of things that are required to be reviewed in order to have the PR approved
+List here all the items you have verified BEFORE sending this PR. Please DO NOT remove any item, striking through those that do not apply. (Just in case, strikethrough uses two tildes. ~~Scratch this.~~)
+-->
+- [ ] I have signed the [Contributor License Agreement](https://www.elastic.co/contributor-agreement/).
+<!--
+We are not asking you to assign copyright to us, but to give us the right to distribute your code without restriction. We ask this of all contributors in order to assure our users of the origin and continuing existence of the code. You only need to sign the CLA once.
```

Comment:
The CLA specifically does grant Elasticsearch a copyright license. (Section 2 -- Grant of Copyright License).

 Has anybody from legal reviewed this statement? I think we should not attempt to interpret the language of the CLA in a PR template and if we do, we should ensure that legal has signed off on any language.

Reply:
This is really a good point. We just moved those lines from the CONTIRBUTING guide to here. Maybe it's wise to remove it and delegate the explanation to the CLA itself

Reply:
Sounds good. I made a corresponding update to the CONTRIBUTING document so everything is aligned: https://github.com/elastic/apm-server/pull/3292

Reply:
Won't that PR be conflicting with this one? Here we remove that paragraph

Reply:
Ah, I didn't see that. I'll close mine then. Thanks!